packages/shared/styles/components/SfGroupedProduct.scss

```diff
@@ -81,4 +81,7 @@ $grouped-product-item__image-background-color: $c-light !default;
       }
     }
   }
+  &--disabled {
```

Comment:
@aniamusial disable is not enough :) 

![Zrzut ekranu 2020-02-1 o 18 37 27](https://user-images.githubusercontent.com/12138170/73596377-f4b59680-4521-11ea-9982-3e056a4b6b81.png)